RealmSwift-swift2.2/Realm.swift

```diff
@@ -96,7 +96,8 @@ public final class Realm {
     // MARK: Transactions
 
     /**
-     Performs actions contained within the given block inside a write transaction.
+     Performs actions contained within the given block inside a write transaction. If the block throws an exception, the
```

Comment:
First line of the docs should always be a (brief) single sentence description of the function, and all addition details go in their own paragraphs.